### DISCUSSION TITLE
Best way to import a local .json file in Nuxt 3 (using Bridge)

### DISCUSSION THREAD

**Discussion by nathanchase** [nuxt/nuxt#18203] [Questions]
I have my Nuxt 2 app building using Bridge, but I have an error relating to importing a .json file.
```
Module parse failed: Unexpected token i in JSON at position 0 while parsing near 'import { watch } fro...' 
File was processed with these loaders:
 * ./node_modules/unplugin/dist/webpack/loaders/transform.js
 * ./node_modules/unplugin/dist/webpack/loaders/transform.js
You may need an additional loader to handle the result of these loaders.
```
In my .vue file, I am importing some dummy data (rather than calling out to an API):
`import { test } from '~/static/testdata/test.json';`

The .json is valid, and works fine in Nuxt 2 to be imported this way. Is there some special consideration for loading .json files directly this way?

**Comment by lucpotage** [comment]
Here is the solution:

```vue
<script setup>
const { test } = await import('~/assets/testdata/test.json')
</script>
```

**Comment by nathanchase** [comment]
ESLint complains about the lack of `async`: 
`Parsing error: Cannot use keyword 'await' outside an async function`

**Comment by misaon** [comment]
@pi0 Is there a solution to this? I would like to call `await Promise.all ([...])` in the setup method due to `await` I can't use it. 😢
`async setup()` doesn't work either and when i try use `useFetch/useAsync` from `@nuxtjs/composition-api` (described in docs), webpack build will fail.

![image](https://user-images.githubusercontent.com/15806356/140948434-26640fd1-cb9b-4a97-9a6e-d726cffe6a92.png)
